### PR TITLE
Exploring the data page

### PR DESCRIPTION
**Pull request by palpatel0**
Hello,
Please find attached my intro page for the book. Feel free to add anything you think is missing or make suggestions.
Merci

### CODE REVIEW

**Comment by TomWFox** [comment]
Can you add the 0d2d75bf-6765-40e0-a0de-1ab8350f4b02.png file to the assets folder please?

**Comment by palpatel0** [comment]
> Can you add the 0d2d75bf-6765-40e0-a0de-1ab8350f4b02.png file to the assets folder please?

done!

**Review by TomWFox — CHANGES_REQUESTED** [review]
A few changes needed, but looking pretty good :)

File: file.Rmd [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+In the `borough` data set there is an additional 'London Heathrow and London City Airports' value, while in the `ward` data set there is an 'Aviation Security (SO18)' value. The implications of this are discussed further on in the notebook.
+
+
+## Categories and Offences
+There are 12 categories, and 56 offences across those categories.
+
+```{python}
+list(data.borough.index.get_level_values(0).unique().sort_values())
+```
```
Comment (palpatel0) [inline file.Rmd:54]: Need help with doing this due to multi-index please @TomWFox

**Review by palpatel0 — COMMENTED** [review]
need help

File: file.Rmd [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+             'boroughs_ward': list(data.ward.index.get_level_values(4).unique().sort_values())}
+bors = pd.DataFrame(data=bor_names)
+bors
+```
+
+All the datasets contain a borough column, which for the `ward` and `lsoa` datasets refer to the borough in which the ward or LSOA sit.
+
+The dataframe above shows the values in the borough columns for the `borough` and `ward` datasets. As you can see here, there are *33* boroughs in each of the data frames, instead of _32_.
+
+In the `borough` data set there is an additional 'London Heathrow and London City Airports' value, while in the `ward` data set there is an 'Aviation Security (SO18)' value. The implications of this are discussed further on in the notebook.
```
Comment (TomWFox) [inline file.Rmd:46]: This is a great improvement, thanks!

File: file.Rmd (hunk 1 above)
Comment (TomWFox) [reply file.Rmd]: CleanShot 2023-12-19 at 22 14 24@2x

I've done it like this, it's not perfect but I think it does the job nicely. Okay with you?

**Review by TomWFox — DISMISSED** [review]
Good to merge!

**Review by TomWFox — DISMISSED** [review]

**Review by TomWFox — APPROVED** [review]
Just added the rose-pine style as well, for consistency, hope that's okay!

**Pull request merged**